graphics/graphviz: Fails to build

Linking the _dot_ tool fails:
```
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: warning: libxdot.so.4, needed by ../../lib/gvc/.libs/libgvc.so, not found (try using -rpath or -rpath-link)
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: warning: libcdt.so.5, needed by ../../lib/gvc/.libs/libgvc.so, not found (try using -rpath or -rpath-link)
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: warning: libpathplan.so.4, needed by ../../lib/gvc/.libs/libgvc.so, not found (try using -rpath or -rpath-link)
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `Dttree'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/cgraph/.libs/libcgraph.so: undefined reference to `Dtlist'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `Pshortestpath'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/cgraph/.libs/libcgraph.so: undefined reference to `dtdisc'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/cgraph/.libs/libcgraph.so: undefined reference to `dtrenew'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `Dtoset'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `freePath'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `Dtobag'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `dtclose'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `dtopen'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `Proutespline'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/cgraph/.libs/libcgraph.so: undefined reference to `dtmethod'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `Dtqueue'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `dtflatten'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `dtsize'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/gvc/.libs/libgvc.so: undefined reference to `make_polyline'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/cgraph/.libs/libcgraph.so: undefined reference to `dtrestore'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/cgraph/.libs/libcgraph.so: undefined reference to `dtextract'
/bob/7b069006fe010a6affc9f4f6a844b9f27b71ee6b/workspace/usr/bin/../lib/gcc/x86_64-cross-linux-gnu/9.2.0/../../../../x86_64-cross-linux-gnu/bin/ld: ../../lib/cgraph/.libs/libcgraph.so: undefined reference to `dtview'
collect2: error: ld returned 1 exit status
```

--- a/recipes/graphics/graphviz.yaml
+++ b/recipes/graphics/graphviz.yaml
@@ -1,7 +1,7 @@
-inherit: [autotools]
+inherit: [cmake, python3]
 
 metaEnvironment:
-    PKG_VERSION: "2.44.1"
+    PKG_VERSION: "13.1.2"
 
 depends:
     - libs::cairo-dev
@@ -18,17 +18,20 @@ depends:
 
 checkoutSCM:
     scm: url
-    url: https://www2.graphviz.org/Packages/stable/portable_source/graphviz-${PKG_VERSION}.tar.gz
-    digestSHA256: 8e1b34763254935243ccdb83c6ce108f531876d7a5dfd443f255e6418b8ea313
+    url: https://gitlab.com/graphviz/graphviz/-/archive/${PKG_VERSION}/graphviz-${PKG_VERSION}.tar.bz2
+    digestSHA256: c062e7ef870b2b2d0196076b601839316ede3a5a10efad2e9d9f2d3fbd9d57ca
     stripComponents: 1
 
+buildTools: [bison, flex]
 buildScript: |
-    export ac_cv_prog_PS2PDF="true"
-    export ac_cv_prog_GROFF="true"
-    autotoolsBuild $1
+    cmakeBuild $1 \
+      -DWITH_GVEDIT=OFF \
+      -DCMAKE_SKIP_INSTALL_RPATH=ON \
+      -DCMAKE_BUILD_RPATH_USE_ORIGIN=ON \
+      -Dwith_sfdp=OFF
 
 packageScript: |
-    autotoolsPackageTgt
+    cmakePackageTgt
 
 provideDeps: ["*-tgt"]
 

--- a/tests/buildall/recipes/all.yaml
+++ b/tests/buildall/recipes/all.yaml
@@ -10,8 +10,7 @@ depends:
     - core::tz-data
     - core::tz-zic
 
-    #  FIXME: See https://github.com/BobBuildTool/basement-gnu-linux/issues/5
-    # - graphics::graphviz
+    - graphics::graphviz
     - graphics::fonts::dejavu
     - graphics::fonts::unicode
     - graphics::imagemagick

